networks/monza/monza-config/src/lib.rs

```diff
@@ -1,19 +1,19 @@
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Config {
-    pub execution_config : monza_execution_util::config::Config,
+    pub execution_config : movement_execution_util::config::Config,
 }
```

Comment:
Should the directory for this file still be `monza/monza-config/`? I think we want it renamed to `networks/suzuka/suzuka-config/src/lib`, or are we keeping these monza dir names? Seems confusing if we kept them.

Reply:
If I understood @l-monninger correctly, there is to be the common part of configuration, now in `movement_execution_util`, composed with the protocol-specific configuration like here in `monza_config`.

Reply:
Yeah, that util will be shared across units using the `movement-opt-executor` -> `maptos-opt-executor`.